profile/profile.yaml

```diff
@@ -0,0 +1,135 @@
+# Preffered name (Characters only)
```

Comment:
please run a grammar and spelling check for the entire doc.. I have noticed several typos